spring-boot-starter/spring-boot-starter-registry/src/main/java/io/servicecomb/springboot/starter/registry/RegistryIntializer.java

```
@@ -42,6 +42,8 @@ public static void initRegistry() {
       } catch (Exception e) {
         LOG.error("init registry error.", e);
       }
+    } else {
+      LOG.info("rest address is null.Service is not registered to service center");
```

It's better to throw the exception out instead of writing some log messages.